spec/PeridicBackgroundSync-index.bs

```diff
@@ -0,0 +1,95 @@
+<pre class='metadata'>
```

Comment:
Can you fix the file name? Peridic -> Periodic

Reply:
done.